pkg/capr/configserver/server.go

```diff
@@ -35,18 +36,22 @@ import (
 )
 
 const (
-	ConnectClusterInfo    = "/v3/connect/cluster-info"
-	ConnectConfigYamlPath = "/v3/connect/config-yaml"
-	ConnectAgent          = "/v3/connect/agent"
+	ConnectClusterInfo          = "/v3/connect/cluster-info"
+	ConnectConfigYamlPath       = "/v3/connect/config-yaml"
+	ConnectClusterAgentYamlPath = "/v3/connect/cluster-agent/{token}.yaml"
```

Comment:
Could you explain a bit about how the new endpoint will be used?

Reply:
`connectClusterAgentYAML` returns a cattle-cluster-agent manifest for the given bearer token, this path is called by the agent manifest script sent via planner https://github.com/rancher/rancher/pull/47620/files#diff-955ac5704543565a62e69b77145857cdbbde42acdb4b154c879e899097302426R348